src/main/java/eu/enmeshed/model/request/LocalRequestResponse.java

```diff
@@ -15,6 +15,6 @@
 @Setter
 @SuperBuilder
 public class LocalRequestResponse extends ContentWrapper<Response> {
-  ZonedDateTime createdAt;
-  RequestResponseSource source;
+  private ZonedDateTime createdAt;
```

Comment:
Why was it added here that these must be private properties compared to the properties of other structures (for example, `LocalRequestSource` of LocalRequestSource.java)?

Reply:
I just added it where I found it. In general it should be everywhere.